### DISCUSSION TITLE
I uninstalled brew on linux mint but my machine keeps trying to install and use it during installations

### DISCUSSION THREAD

**Discussion by thamnasc** [Homebrew/discussions#3361] [Linux] [answered]
I already use the following command:
`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"`
I also already deleted the linuxbrew directory, which was empty after the uninstall process, and updated the .profile file.

I am trying to install ruby using rvm, but during the installation, my machine keeps trying to install homebrew. I don't want to use it anymore. 

This keep happening:

$ rvm install ruby-3.0.1
Searching for binary rubies, this might take some time.
No binary rubies available for: mint/20.3/x86_64/ruby-3.0.1.
Continuing with compilation. Please read 'rvm help mount' to get more information on binary rubies.
Checking requirements for osx_brew.
About to install Homebrew in the default location `/usr/local`.

It is possible to select a custom location, however it is not recommended and some things might not work.
You should do it only if you do not have write rights to `/usr/local`.

Press ENTER to install Homebrew in the default location `/usr/local`
or type a custom path (needs to be writable for the current user)
:

**Comment by Bo98** [comment, marked as answer]
We don't ship RVM or touch it in any way so this seems like an RVM configuration problem.

Check `rvm autolibs status`. You might need to reset those settings with `rvm autolibs reset`.

**Reply by gromgit** [reply]
I don't use RVM, but:
```
Checking requirements for osx_brew.
About to install Homebrew in the default location /usr/local.
```
@thamnasc , did you copy your RVM setup from a previous macOS installation? That may explain why it's trying to do a macOS Homebrew install on a Linux box.

**Reply by thamnasc** [reply]
> We don't ship RVM or touch it in any way so this seems like an RVM configuration problem.
> 
> Check `rvm autolibs status`. You might need to reset those settings with `rvm autolibs reset`.

It solved my problem. Thanks everyone!
